### PR TITLE
elfloader: Replace calls to reset_cntvoff

### PR DESCRIPTION
**Pull request by kent-mcleod**
These calls can now be implemented via binding the /timer driver in the elfloader's device tree configuration.

Test with: seL4/seL4_tools#168

### CODE REVIEW

File: src/plat/tx1/overlay-tx1.dts
Comment (Indanz) [inline src/plat/tx1/overlay-tx1.dts:12]: For consistency's sake, timer should be after psci.

**Review by Indanz — APPROVED** [review]
I haven't looked at the Elfloader changes yet, but I assume this is needed there.

**Comment by kent-mcleod** [comment]
Need approval on https://github.com/seL4/seL4_tools/pull/168 before this can be merged.

**Pull request merged**
